src/coldcard.test.js

```diff
@@ -0,0 +1,175 @@
+import {ColdcardMultisigWalletConfig, WALLET_CONFIG_VERSION} from './coldcard';
+
+describe("ColdcardMultisigWalletConfig", () => {
+  const jsonConfigUUID = {
+    "name": "Test",
+    "addressType": "P2SH",
+    "network": "testnet",
+    "quorum": {
+      "requiredSigners": 2,
+      "totalSigners": 3,
+    },
+    "startingAddressIndex": 0,
+    "extendedPublicKeys": [
+      {
+        "name": "unchained",
+        "xpub": "tpubDF17mBZYUi35r7UEkGr7SjSisec8QL2J1zGh9WQtmnhJtHeMFy2eH2Xsnr2ynxENbqHmcEA4NnoT8T6RZxks4G5evZdWy1RbSPTm8LtNPU3",
+        "bip32Path": "Unknown",
+        "xfp": "83471e78",
+      },
+      {
+        "name": "Os_words1",
+        "xpub": "tpubDF61GHbPYRhEEsqNTDF2jbMBkEMmoksy1h2URbhZ8p7JfR9QRgbn6vkeA7g3t4Ue6uSHhYJxD9mRVz1ZQVYyW3RAPPuwVM4UeZyZPKu89DY",
+        "bip32Path": "m/45'/1'/50'/0",
+        "xfp": "f57ec65d",
+      },
+      {
+        "name": "Os_words2",
+        "xpub": "tpubDEZxsxeamoPim6T1hg2igm4vDmdQrQnHVH5gM5NjcjowtgYVv5ZhhR5sFgRVNjRFGk1HmxsFsYhu3jGaAGVCpCsL5AbAVk6xKssr6gK3tPk",
+        "bip32Path": "m/45'/1'/60'/0",
+        "xfp": "f57ec65d",
+      },
+    ],
+    "uuid": "OWPyFOA1"
+  };
+  const jsonConfigName = {
```

Comment:
Any reason why these JSON objects need to be duplicated almost exactly? Since there's already some parsing/stringifying and copying below, couldn't there just be one as a base that is adjusted as necessary for each specific test?